### ISSUE TITLE
Grid viewer is not accessible to narrator at all

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS#3173]
View(mtcars)
Activate grid view window
Try to navigate through fields with keyboard
- no focus indication, narrator is silent

**Comment by MikhailArkhipov** [comment]
There are significant issues here since grid is composed of 3 visual grids (row header, column header and the data section) and each of those consists of visuals and not UI elements. This confuses readers since they rely on a visual tree. Even when we provide automation peers to the visual grid inspect and narrator do not pick them up. Standard DataGrid (as in import data preview) works since all elements are UI elements and DG is a container of rows which are containers of elements (ItemsControl).

**Comment by MikhailArkhipov** [comment]
Won't fix. Will use some commercial control next major release
